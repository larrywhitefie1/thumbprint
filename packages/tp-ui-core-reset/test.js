const sass = require('node-sass');

it('compiles correctly', () => {
    const css = sass.renderSync({ file: 'packages/tp-ui-core-reset/_index.scss' }).css.toString();
    expect(css).toMatchSnapshot();
});
