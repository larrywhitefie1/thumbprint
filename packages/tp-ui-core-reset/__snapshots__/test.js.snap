// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compiles correctly 1`] = `
"html {
  box-sizing: border-box; }

*,
*:before,
*:after {
  box-sizing: inherit; }

body {
  margin: 0; }

h1,
h2,
h3,
h4,
h5,
h6,
p,
blockquote,
pre,
dl,
dd,
ol,
ul,
form,
fieldset,
legend,
table,
th,
td,
figure,
caption,
hr {
  margin: 0;
  padding: 0; }

ol,
ul {
  list-style: none; }

h1,
h2,
h3,
h4,
h5,
h6 {
  font-weight: 400;
  font-size: inherit; }

table {
  border-collapse: collapse;
  border-spacing: 0; }

abbr[title],
dfn[title] {
  cursor: help; }

u,
ins {
  text-decoration: none; }

ins {
  border-bottom: 1px solid; }

img {
  font-style: italic; }

button,
select,
option,
input[type='checkbox'] + label,
input[type='radio'] + label {
  cursor: pointer; }

[type='text']:active,
[type='text']:focus,
[type='password']:active,
[type='password']:focus,
[type='email']:active,
[type='email']:focus,
[type='number']:active,
[type='number']:focus,
[type='search']:active,
[type='search']:focus,
[type='url']:active,
[type='url']:focus,
textarea:active,
textarea:focus {
  cursor: text;
  outline: none; }

[type='text'],
[type='password'],
[type='email'],
[type='number'],
[type='search'],
[type='url'],
textarea {
  -webkit-appearance: none; }

a {
  text-decoration: none; }

a:active,
a:hover {
  outline: 0; }

sub,
sup {
  font-size: 100%;
  vertical-align: baseline; }

em {
  font-style: italic; }

b,
strong {
  font-weight: 700; }

fieldset {
  border: 0; }

textarea {
  vertical-align: top; }

button[disabled],
html input[disabled],
select[disabled],
textarea[disabled] {
  cursor: default; }

button,
select {
  text-transform: none; }

button,
input,
select,
textarea {
  font-family: inherit;
  font-size: 100%;
  margin: 0; }

button::-moz-focus-inner {
  padding: 0;
  border: 0; }

input[type='search']::-webkit-search-cancel-button,
input[type='search']::-webkit-search-decoration {
  -webkit-appearance: none; }

button,
html input[type='button'],
input[type='reset'],
input[type='submit'] {
  -webkit-appearance: button;
  cursor: pointer; }

[required] {
  box-shadow: none; }
"
`;
